roles/st2repos/tasks/debian.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Install prereqs (Debian)
+  become: yes
+  apt:
+    name: "{{ item }}"
+    state: present
+    update_cache: yes
```

Comment:
This will break Ansible Idempotence as @bigmstone discovered before, since it will `apt-get upgrade` every time.